sql/hathifiles.sql

```diff
@@ -0,0 +1,47 @@
+-- Used by HathifilesDatabaseVerifier
+
+CREATE TABLE IF NOT EXISTS `hf` (
+  `htid` varchar(255) NOT NULL,
```

Comment:
Probably should not allow nulls here (but really this is the responsibility of hathifiles-database)

Reply:
To address later.